pkg/filterengine/filters/job_status_checker.go

```diff
@@ -0,0 +1,51 @@
+// JobStatusChecker filter to send notifications only when job succeeds
+// and ignore other update events
+
+package filters
+
+import (
+	"github.com/infracloudio/botkube/pkg/config"
+	"github.com/infracloudio/botkube/pkg/events"
+	"github.com/infracloudio/botkube/pkg/filterengine"
+	log "github.com/infracloudio/botkube/pkg/logging"
+
+	batchV1 "k8s.io/api/batch/v1"
+)
+
+// JobStatusChecker checks job status and adds message in the events structure
+type JobStatusChecker struct {
+}
+
+// Register filter
+func init() {
+	filterengine.DefaultFilterEngine.Register(JobStatusChecker{})
+}
+
+// Run filers and modifies event struct
+func (f JobStatusChecker) Run(object interface{}, event *events.Event) {
+	// Run filter only on Job update event
+	if event.Kind != "Job" && event.Type != config.UpdateEvent {
+		return
+	}
+	jobObj, ok := object.(*batchV1.Job)
+	if !ok {
+		return
+	}
+
+	// Check latest job conditions
+	jobLen := len(jobObj.Status.Conditions)
+	if jobLen < 1 {
+		event.Skip = true
+		return
+	}
+	c := jobObj.Status.Conditions[jobLen-1]
+	if c.Type == batchV1.JobComplete {
+		event.Messages = []string{"Job succeeded!"}
```

Comment:
Should we include job name or some sort of unique identifier?

Reply:
@ssudake21 Job name, namespace and cluster name can be seen in the notification
![image](https://user-images.githubusercontent.com/7098659/60071817-0e476200-9739-11e9-8e66-acda98a4e142.png)